_posts/2020-01-26-newpipe-0.18.2-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube-Extractor"
+short: "NewPipe 0.18.1 released"
```

Comment:
The version is `.2`, you forgot to update this line.

Reply:
I did